app/src/androidTest/java/ch/epfl/polychef/images/ImageHandlerTest.java

```diff
@@ -41,13 +41,17 @@
     @Mock
     UploadTask task;
 
+    @Mock
+    ImageStorage mockImageStorage;
+
     ImageHandler realImageHandler;
     ImageHandler fakeImageHandler;
 
     @Before
     public void setImagesAndMock() {
         MockitoAnnotations.initMocks(this);
         when(task.addOnSuccessListener(any())).thenReturn(task);
+        when(mockImageStorage.upload(any(), any(), any(), any())).thenReturn(task);
```

Comment:
Don't use raw `any()` instead specify the class of the element (i.e. `.upload(any(byte[].class), anyString(), anyString(), anyString())`)

Reply:
You are right, I'll change that